language-examples/java/auth/src/main/java/com/vectara/auth/JwtFetcher.java

```diff
@@ -102,7 +101,7 @@ public String fetchClientCredentialsJwt() {
         return null;
       }
       if (map.containsKey("access_token")) {
-        return JWT.decode(String.valueOf(map.get("access_token"))).getToken();
+        return (String) map.get("access_token");
```

Comment:
It's more robust to just say:
```
String.valueOf(map.get("access_token"));
```
This code cannot crash for any reason.

Reply:
done